libraries/AP_RangeFinder/RangeFinder.h

```diff
@@ -32,6 +32,7 @@ class RangeFinder
 {
 public:
     friend class AP_RangeFinder_Backend;
+    friend class AP_RangeFinder_analog; // bodgy detect function
```

Comment:
Not needed? Looks like the *bodgy detect function* is fixed in the second commit.